src/lib/snakeCase.js

```diff
@@ -0,0 +1,37 @@
+const snakeCase = (startingString) => {
+
+  var array = startingString.split('');
+  var newArray = [];
+
+  for (let i = 0; i < array.length; i++){
+
+    if(i === 0){
```

Comment:
Consider moving this `if` statement outside of the for loop, and starting the for loop at `i = 1` - if we do this, we gain a (very minor) performance increase in that the `for` loop body will not need to perform this unnecessary comparison.